couchbase-entity/src/main/java/com/kaufland/documentation/DocumentationGenerator.kt

```diff
@@ -73,7 +73,8 @@ class DocumentationGenerator(path: String, fileName: String) {
             return
         }
 
-        docuEntitySegments[entityHolder.sourceClazzSimpleName] = div().with(h1(entityHolder.sourceClazzSimpleName), evaluateAvailableTypes(entityHolder.sourceElement), br(), *buildComment(entityHolder.comment), br(), table(attrs(".table"), thead(*createTableHead()), tbody(
+        val btnWithSectionLink = "<button onclick=\"alert(window.location.protocol + '//' + window.location.host + window.location.pathname + window.location.search + '#${entityHolder.sourceClazzSimpleName}');\">showLink</button>"
```

Comment:
I'm not sure what the button with the alert popup is supposed to do :thinking: Did you want to create a table of content?

Reply:
no I just added sections to the documentation and this alert displays the url of that section. This could be useful to refer to a specific document